en_US.ini

```diff
@@ -598,10 +598,10 @@ Force real clock sync (slower, less lag) = Force real clock sync (slower, less l
 General = General
 Help the PPSSPP team = Help the PPSSPP team
 Host (bugs, less lag) = Host (bugs, less lag)
-I/O on thread (experimental) = I/O on thread (experimental)
+I/O on thread (experimental) = I/O on thread
 IO timing method = I/O timing method
 MMDDYYYY = MMDDYYYY
-Multithreaded (experimental) = Multithreaded (experimental)
+Multithreaded (experimental) = Multithreaded (glitches)
```

Comment:
I wish there was a list of those, most of them you would need either an unbalanced PC setup or a weak Android device to spot them. You may need to state that "it may cause the opposite effect" and at the same time that "it may break a game".